yorkie/src/main/kotlin/dev/yorkie/document/Document.kt

```diff
@@ -308,6 +308,7 @@ public class Document(public val key: Key) {
         public data class ChangeInfo(
             public val message: String,
             public val operations: List<OperationInfo>,
+            public val actorID: ActorID? = null,
```

Comment:
do we need this to be nullable?
JAVA version of the `ChangeInfo` protobuf model seems have a non-null `actorID`

Reply:
I just ported the JS SDK, but it seems correct to set it as non-null. 
the android SDK implements `Change` to always have `ChangeID` and its `actorID`, so `ChangeInfo` will have a non-null `actorID`.
[0a0c396](https://github.com/yorkie-team/yorkie-android-sdk/pull/109/commits/0a0c3964503573bda40c0ae4b9ea3db6cadfd24d)